src/components/map/layers/mapIcons/NodeMarker.tsx

```diff
@@ -197,7 +197,7 @@ class NodeMarker extends Component<INodeMarkerProps> {
                     && this.onMoveMarker('coordinates')}
                 >
                 {
-                    displayCircle
+                    displayCircle && Boolean(this.props.node.stop!.radius)
```

Comment:
Hmm, if node has no radius, before it displayed circle with default radius?

I would refactor a bit and say:

renderStopRadiusCircle()

```
    private renderStopRadiusCircle = () => {
        const stop = this.props.node.stop;
        if (!stop) throw new Error('Stop not found'); // Should never occur
        const radius = stop.radius ? stop.radius : DEFAULT_RADIUS;
        return (
            <Circle
                className={s.stopCircle}
                center={this.props.node.coordinates}
                radius={radius}
            />
        );
    }
```